src/PIL/ImageFile.py

```diff
@@ -63,15 +63,19 @@
 # Helpers
 
 
-def raise_oserror(error):
+def _get_oserror(error, encoder=False):
```

Comment:
Let's make `encoder` keyword only, because calling `_get_oserror(errcode, encoder=True)` is clearer than `_get_oserror(errcode, True)`:


```suggestion
def _get_oserror(error, *, encoder=False):
```

Reply:
Done.